internal/common/db.go

```diff
@@ -212,6 +212,16 @@ func GetHostFromDB(db *gorm.DB, infraEnvId, hostId string) (*Host, error) {
 	return &host, nil
 }
 
+func GetClusterHostFromDB(db *gorm.DB, clusterId, hostId string) (*Host, error) {
```

Comment:
Where is this used?

Reply:
currently preparation for mazgan, we will use on the call when we need to find host by its cluster and not by infra env